index.html

```diff
@@ -2031,6 +2051,62 @@ <h2>
           Visual representation of a [=Standard Gamepad=] layout.
         </figcaption>
       </figure>
+      <section data-dfn-for="ExtendedMapping" data-link-for="ExtendedMapping">
+        <h3>
+	  <dfn>ExtendedMapping</dfn>
+        </h3>
+        <p>
```

Comment:
Oh wait. ExtendedMapping is not an IDL thing... it's a concept. 

```suggestion
      <section>
        <h3>
	  <dfn>Extended Gamepad Mapping</dfn>
        </h3>
        <p>
```